zerver/lib/openapi.py

```diff
@@ -38,7 +38,7 @@ def spec(self) -> Dict[str, Any]:
         last_modified = os.path.getmtime(self.path)
         # Using != rather than < to cover the corner case of users placing an
         # earlier version than the current one
-        if self.last_update != last_modified:
+        if self.last_update != last_modified or self.data is None:
```

Comment:
Is it possible for `last_modified` to be `None`?  Otherwise, I don't see how this would come up.